plugins/pulp_python/plugins/distributors/steps.py

```diff
@@ -76,14 +80,22 @@ def process_main(self):
             body = ElementTree.SubElement(html, 'body')
             # For each package, we need to make a reference in index.html and also make a directory
             # with its own index.html for the package
-            for name, packages in packages.items():
+            for name, packages in projects.items():
                 element = ElementTree.SubElement(body, 'a', {'href': name})
                 element.text = name
                 ElementTree.SubElement(body, 'br')
                 PublishMetadataStep._create_package_index(name, simple_path, packages)
 
             index.write(ElementTree.tostring(html, 'utf8'))
 
+        for name, packages in projects.items():
+            project_metadata_path = os.path.join(api_path, name, 'json')
+            os.makedirs(project_metadata_path)
+            project_index_metadata_path = os.path.join(project_metadata_path, 'index.html')
```

Comment:
Oof. It's confusing to have a file that's called index.html that just contains JSON.